src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -161,19 +164,21 @@ def update_issue_impact_labels(testcase, issue):
 
   if testcase.regression.startswith('0:'):
     # If the regression range starts from the start of time,
-    # then we assume that the bug impacts stable.
-    new_impact = data_types.SecurityImpact.STABLE
+    # then we assume that the bug impacts extended stable.
+    new_impact = data_types.SecurityImpact.EXTENDED_STABLE
```

Comment:
Also: do we need to give a head's up to any PMs before we do this? 

They may have scripts today that depend on `Security_Impact-Stable` and not yet ExtendedStable.

Reply:
Reversed this piece of code as there are some scripts reply this label.